### PR TITLE
structured data operators for serializing tablerows, triples and keyvalue pairs added

### PR DESCRIPTION
**Pull request by csrajmohan**
* Added new serialization operators for structured data: Table rows, Triples & KeyValue pairs
* Renamed table_operators -> struct_data_operators so that it best captures all structured data operators like the ones for Tables, Triples etc. and makes better sense to add other structured data operators to this in future.
* Added 3 new datatask cards to validate new operators
- wiki_bio : KeyValue Pair to Text(Data2Text task), Uses KeyValuePairs Serializer
- dart : Triples to Text(Data2Text task), Uses TriplesSerializer
- tablerow_classify : Table row classification using a Kaggle dataset(Heart disease prediction), Uses TableRowSerializer

### CODE REVIEW

File: prepare/cards/tablerow_classify.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    ),
+    preprocess_steps=[
+        SplitRandomMix(
+            {"train": "train[70%]", "validation": "train[10%]", "test": "train[20%]"}
+        ),
+        RenameFields(field_to_field={"HeartDisease": "label"}),
+        MapInstanceValues(mappers={"label": {"0": "Normal", "1": "Heart Disease"}}),
+        AddFields(
+            fields={
+                "text_type": "Person",
```
Comment (elronbandel) [inline prepare/cards/tablerow_classify.py:26]:
```suggestion
                "text_type": "person medical record",
```
I think it is more descrptive, no?

File: prepare/tasks/generation.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["input"],
+        inputs=["input", "type_of_input"],
```
Comment (elronbandel) [inline prepare/tasks/generation.py:6]: Maybe would be also good to have type of output?
```suggestion
        inputs=["input", "type_of_input", "type_of_output"],
```

**Review by elronbandel — CHANGES_REQUESTED** [review]
Overall this looks really good. I really appreciate that you use the existing tasks and even improve them!!  Go over my comment and ask for review when done and we can get it merged!

**Comment by elronbandel** [comment]
In case its needed: the test is currently failing because of import from `unitxt` rather then `src.unitxt`

File: prepare/cards/tablerow_classify.py (hunk 1 above)
Comment (csrajmohan) [reply prepare/cards/tablerow_classify.py]: Done

File: prepare/tasks/generation.py (hunk 2 above)
Comment (csrajmohan) [reply prepare/tasks/generation.py]: Done

**Comment by csrajmohan** [comment]
import for test file fixed

**Review by elronbandel — APPROVED** [review]
Great contribution!

**Pull request merged**